MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range"
+                        RangeSelectionDirection="Forward">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+this.Calendar.RangeSelectionDirection = CalendarRangeSelectionDirection.Forward;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Forward range selection in .NET MAUI Calendar.](images/selection/maui-forward-range-selection.png)
+
+#### Range Selection Direction - Backward
+If the `RangeSelectionDirection` property to `Backward` allows you to select only the dates before the selected range end date and the dates after the range end date is considered as disabled date. The end date will not be changed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range"
+                        RangeSelectionDirection="Backward">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+this.Calendar.RangeSelectionDirection = CalendarRangeSelectionDirection.Backward;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Backward range selection in .NET MAUI Calendar.](images/selection/maui-backward-range-selection.png)
+
+#### Range Selection Direction - Both
+If the `RangeSelectionDirection` property to `Both` allows you to extend the selection from the selected range. Then the tapped date is considered based on (if the date is nearby start date then start date is updated, else if the date is nearby end date then end date is updated, else if in-between the start and end date and both have same distance then the start date is updated.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range"
+                        RangeSelectionDirection="Both">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+this.Calendar.RangeSelectionDirection = CalendarRangeSelectionDirection.Both;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Both Range selection in .NET MAUI Calendar.](images/selection/maui-range-selection.png)
+
+#### Range Selection Direction - None
+If the `RangeSelectionDirection` property to `None` allows you to select only the single range and the dates after the range end date and before start date is considered as disabled date. It will remain in the initial range.
```

Comment:
If you set the `RangeSelectionDirection` property to `None,` you can select only the single range, while dates after the range end date and before the start date are considered disabled. It will remain in the initial range.

Reply:
Content modified